html/parking/index.html

```diff
@@ -0,0 +1,3 @@
+<h1>Parking</h1>
+
+<p>You're in parking bro...</p>
```

Comment:
You are in parking bro ... keep swimming forward and breathe so you don't get eaten by sharks.